ProcessMaker/processmaker
# Remove External Integrations Setting Due to CDATA Package Deprecation

This PR removes the "External Integrations" option in the settings table, as it is no longer needed following the deprecation of the CDATA Package.

## Changes
- Added a migration to remove the settings related to "External Integrations" (CDATA).

## How to Test
1. Ensure the `docker-executor-cdata` and `package-cdata` are installed.
2. Confirm the "External Integrations" setting exists in the database and UI.
3. Run `php artisan migrate`.
4. Verify the "External Integrations" setting is removed from the database and UI.
5. Run `php artisan migrate:rollback --step=1`
6. Verify the "External Integrations" setting is restored in the database and UI.

## Related Tickets & Packages
- [FOUR-21298](https://processmaker.atlassian.net/browse/FOUR-21298)

## Code Review Checklist


[FOUR-21298]: https://processmaker.atlassian.net/browse/FOUR-21298?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
